crates/services/gas_price_service/src/v1/da_source_adapter/block_committer_costs.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(non_snake_case)]
```

Comment:
you can remove this line and add `#[allow(non_snake_case)]` just before the `mod tests` declaration

Reply:
addressed in d562d720fc